specs/vep-994-jupyter-notebook-integration/README.md

```diff
@@ -76,36 +76,29 @@ Legend (terms and table are based on Pragmatic Marketing recommendations):
 JupyterLab is the next-generation user interface for Project Jupyter offering all the familiar building blocks of the classic Jupyter Notebook (notebook, terminal, text editor, file browser, rich outputs, etc.) in a flexible and powerful user interface.
 Fundamentally, JupyterLab is designed as an extensible environment. JupyterLab extensions can customize or enhance any part of JupyterLab.
 
-The proposed design describes the solution for creating a Jupyter UI for VDK. For this purpose, a JupyterLab extension will be implemented, and it will give us the chance to create graphical elements like buttons and widgets – alternatives of currently used CLI commands. An iPython package will also be implemented to export VDK variables. This package will be imported to JupyterLab, and they will be accessible from there.
+The proposed design describes the solution for creating a Jupyter UI for VDK. For this purpose, a JupyterLab extension will be implemented, and it will give us the chance to create graphical elements like buttons and widgets – alternatives of currently used CLI commands. 
+
+Since JupyterLab works with notebook files the proposed design should support getting the job steps from them. To the purpose, a new VDK plugin will be introduced which will allow vdk to run steps which came from notebook files.
+
+|       Operation        |                                                                                                                                                                                                                          Flow                                                                                                                                                                                                                          | Covered  use cases | Priority |
+|:----------------------:|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------:|:------------------:|:--------:|
+|    Run VDK Jupyter     |                                                                                                                                                                        The user runs a single CLI command which opens the web version of JupyterLab. Examples: vdk jupyter lab                                                                                                                                                                         |         1          |   high   |
```

Comment:
Is it a common practice for jupyter to be started from lets say vdk context, or vice versa -> vdk to be started from jupyter?

Reply:
Starting Jupyter from VDK context would have some benefits over standard "run Jupyter with installed extension". I don't think starting VDK from Jupyter would have benefits, I amnion even sure whether it is possible.